examples/typescript-with-swc/package.json

```diff
@@ -8,9 +8,10 @@
   },
   "devDependencies": {
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.0",
-    "@swc/core": "^1.2.85",
+    "@swc/core": "^1.2.88",
     "@types/react": "^17.0.20",
     "@types/react-dom": "^17.0.9",
+    "core-js": "^3.17.3",
```

Comment:
Is this intentional?

Reply:
Yes. `swc` need explicit `core-js` dependency for compat.